src/deployment/localContainersHelpers.ts

```diff
@@ -321,8 +343,19 @@ export function setLocalContainersFormComponents(
             propertyName: "version",
             label: LocalContainers.selectImage,
             required: true,
-            tooltip: LocalContainers.selectImageTooltip,
+            tooltip:
+                arch() === "arm64"
+                    ? LocalContainers.sqlServer2025ArmErrorTooltip
+                    : LocalContainers.selectImageTooltip,
             options: versions,
+            validate(_state, value) {
+                // Handle ARM64 architecture case where SQL Server 2025 latest is broken
+                const isArm64With2025 = arch() === "arm64" && value.toString().includes("2025");
```

Comment:
Aren't all containers broken in arm64 windows? 
Isn't this error specific to arm64 Macs only?